src/pages/settings/Profile/LoginField.js

```diff
@@ -63,6 +64,18 @@ class LoginField extends Component {
         }
     }
 
+    /**
+    * Return label margin.
+    *
+    * @param {Boolean} addedPhone
+    * @param {Boolean} validatedPhone
+    * @returns {Object}
+    */
+    getLabelMargin(addedPhone, validatedPhone) {
+        return addedPhone && !validatedPhone ? styles.mb0 : {};
```

Comment:
Alternative thought: we can remove the function parameters.
And change the function comment to something like Zero bottom margin for no phone or unverified phone